src/Dockerfile.linux

```diff
@@ -16,12 +16,14 @@ RUN CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -a -o ${NAME}_darwin_amd64
 RUN CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -a -o ${NAME}_darwin_arm64
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o ${NAME}_linux_64
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=386 go build -a -o ${NAME}_linux_32
-# TODO WINDOWS
-# WORKDIR $BASE_DIR/cmd/windows
-# RUN go mod tidy
-# RUN go test -v
+WORKDIR $BASE_DIR/cmd/windows
+RUN go mod tidy
+# can't test windows on linux
+RUN CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -a -o ${NAME}_win_64.exe
+RUN CGO_ENABLED=0 GOOS=windows GOARCH=386 go build -a -o ${NAME}_win_32.exe
```

Comment:
So you are able to build all versions of the binary in either of the containers? Are the outputs  equivalent? It looks like the version that gets bundled just depends on the OS being used to run `mvn compile`, is that right?

Reply:
Yes, the outputs are equivalent, though I haven't done any explicit testing on that...

Reply:
CC @daniel-beck

Reply:
I don't understand the ping. Could you elaborate @jglick ?

Reply:
Sorry. Just about whether builds from sources are guaranteed to produce identical outputs regardless of the build machine’s OS, architecture, and tooling.